README.md

```diff
@@ -12,6 +12,10 @@ It takes as input Data Quality validation tests defined using a flexible and reu
 
 `CloudDQ` currently supports in-place validation of BigQuery data.
 
+* For a further introduction and an explanation of the concepts, please see our [Overview](OVERVIEW.md])
+* For instructions on how to use CloudDQ, see the [User Manual](USERMANUAL.md)
```

Comment:
I feel both Overview and User Manual contain quite a lot, so it might be useful to unpack what's included in them and mention it in this section/paragraph, so users are better informed about what they can find in those docs

Reply:
Add, for both: "For..., including X, Y, Z, see..." with XYZ high level sections.